celestia-node.md

```diff
@@ -0,0 +1,148 @@
+# Running a Celestia Node
+
+- [Running a Celestia Node](#running-a-celestia-node)
+  - [Installation](#installation)
+  - [Full Node Configuration](#full-node-configuration)
+    - [Getting trusted hash](#getting-trusted-hash)
+    - [Running full node](#running-full-node)
+      - [1. Initialize the full node](#1-initialize-the-full-node)
+      - [2. Edit configurations (adding other celestia full nodes)](#2-edit-configurations-adding-other-celestia-full-nodes)
+  - [Light Client Configuration](#light-client-configuration)
+    - [Getting trusted hash](#getting-trusted-hash-1)
+    - [Running Light Client](#running-light-client)
+- [Data Availability Sampling(DAS)](#data-availability-samplingdas)
+  - [Pre-Requisites](#pre-requisites)
+    - [Legend](#legend)
+  - [Steps](#steps)
+
+## Installation
+Make sure that you have `git` and `golang` installed
+```sh
+git clone https://github.com/celestiaorg/celestia-node.git
+make install
+```
+
+
+## Full Node Configuration
+
+### Getting trusted hash
+> Caveat: You need a running celestia-app in order to continue this guideline. Please refer to [celestia-app.md](https://github.com/celestiaorg/networks/celestia-app.md) for installation.
+
+
+You need to have the trusted hash in order to initialize the Celestia full node
+In order to know the hash, you need to query the celestia-app:
+```sh
+curl -s http://localhost:26657/block?height=1 | grep -A1 block_id | grep hash
+```
+
+### Running full node
+#### 1. Initialize the full node
+```sh
+celestia full init --core.remote <ip:port of celestia-app> --headers.trusted-hash <hash_from_celestia_app>
+```
+
+Example:
+```sh 
+celestia full init --core.remote tcp://127.0.0.1:26657 --headers.trusted-hash 4632277C441CA6155C4374AC56048CF4CFE3CBB2476E07A548644435980D5E17
+```
+
+#### 2. Edit configurations (adding other celestia full nodes)
+
+In order for your Celestia full node to communicate with other Celestia full nodes, then you need to add them as `mutual peers` in the `config.toml` file and allow the peer exchange.
```

Comment:
We need a mutual peer file for the users to add more real peers for full node rather then this